### PR TITLE
Elaborating Cron Jobs 

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**
Elaborating Cron Jobs in common areas of concern with web projects.

### CODE REVIEW

File: content/docs/common-areas/cron-jobs.md [hunk 1]
```diff
@@ -9,3 +9,27 @@ description: Cron jobs are a way to schedule and automate tasks on a web server.
...
 
 In order to operate on the web, cron jobs used the web server called a cron daemon. It launches your cron jobs at the appointed times by running in the background. All of your tasks and their times are listed in the configuration file crontab, which is where this schedule is kept. On a server, Cron is a background-running time-based job scheduler.
+
+To set up a cron job, you will typically need to use a command-line interface (CLI) tool. Most web hosting providers offer a simple GUI for setting up cron jobs, but if you have access to the server directly, using the CLI is usually the more powerful and direct approach.
+
+The syntax for setting up a cron job can vary depending on the operating system and server setup. However, the basic format typically includes a schedule for when the task should run (using the `* * * * *` notation, with each asterisk representing a different time interval) and the command that should be executed when the task runs.
+
+For example, to schedule a task to run every day at midnight, the cron job command might look like this:
```
[the remark this answers is not in the document]
Comment (hussainweb) [reply content/docs/common-areas/cron-jobs.md:19]: Can you explain crontab more? "For details..." is not a great explanation of why we want the reader to check that page.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/common-areas/cron-jobs.md [hunk 2]
```diff
@@ -8,4 +8,40 @@ description: Cron jobs are a way to schedule and automate tasks on a web server.
...
-In order to operate on the web, cron jobs used the web server called a cron daemon. It launches your cron jobs at the appointed times by running in the background. All of your tasks and their times are listed in the configuration file crontab, which is where this schedule is kept. On a server, Cron is a background-running time-based job scheduler.
+## Crontab
+
+In Cron Jobs, there is a file called crontab that contains the commands that are run by the cron daemon. The cron daemon is a background process that runs on your server and executes commands at scheduled intervals.
+
+For instance, Drupal uses crontab to run maintenance tasks such as cleaning up log files, checking for updates, and generating sitemaps. These tasks are important for keeping our Drupal site healthy and secure. We have two ways to set up crontab in Drupal:
+
+- Automatic cron: This is the default method, and it uses the cron daemon that is already running on your server. To enable automatic cron, go to Administration > Configuration > System > Cron and set the Frequency to the desired interval.
+
+- Manual cron: This method involves creating a cron job that runs the drush cron command.
```
Comment (hussainweb) [inline content/docs/common-areas/cron-jobs.md:19]: This is not accurate.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/common-areas/cron-jobs.md (hunk 2 above)
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/cron-jobs.md:19]: Please specify the inaccurate part so that I can make changes.

Comment (hussainweb) [reply content/docs/common-areas/cron-jobs.md:19]: The whole section that I have highlighted... It's all inaccurate. It's either the opposite to what you have written or just completely off the track.

Comment (hussainweb) [reply content/docs/common-areas/cron-jobs.md:19]: The description for Manual Cron is fine.

**Review by hussainweb — APPROVED** [review]

**Pull request merged**